<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gobang_internet</class>
 <widget class="QMainWindow" name="gobang_internet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>250</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>800</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="opponentLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="prepareStateLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>未准备</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="ipLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>IP: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="fdLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FD: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="prepareBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>准备</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitRoomBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>580</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出房间</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="regretBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>510</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>悔棋</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loseBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>590</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>认输</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>670</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出游戏</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="LE_recv">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>180</width>
        <height>250</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_send">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(:/res/BackGround2.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="playerLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="msgLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="roomNameLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="refuseBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/res/BackGround2.png)</string>
    </property>
    <property name="text">
     <string>拒绝</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agreeBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/res/BackGround2.png)</string>
    </property>
    <property name="text">
     <string>同意</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blackBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/res/BackGround2.png)</string>
    </property>
    <property name="text">
     <string>先手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="whiteBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/res/BackGround2.png)</string>
    </property>
    <property name="text">
     <string>后手</string>
    </property>
   </widget>
   <widget class="QLabel" name="prepareLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>731</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>请准备....</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="answerLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>751</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>ADSFASDFA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="victoryLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>601</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>黑棋胜利</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="waitLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>781</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>44</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>等待对方响应请求</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
